terraform/plan.go

```diff
@@ -50,6 +58,42 @@ func (p *Plan) String() string {
 	return buf.String()
 }
 
+// Sums returns a PlanSums with sums of resources to be modified
+// based on given Diff
+func (p *Plan) Sums() *PlanSums {
+	sums := &PlanSums{
+		ToChange:           0,
+		ToCreate:           0,
+		ToDestroy:          0,
+		ToDestroyAndCreate: 0,
```

Comment:
These fields default to their zero values, but the extra clarity doesn't hurt. :+1: